simulation_ws/src/sagemaker_rl_agent/markov/s3_boto_data_store.py

```diff
@@ -43,6 +43,22 @@ def get_info(self):
     def undeploy(self) -> bool:
         return True
 
+    def lock(self):
+        try:
+            s3_client = self._get_client()
+            s3_client.upload_fileobj(Fileobj=io.BytesIO(b''),
+                             Bucket=self.params.bucket,
+                             Key=self._get_s3_key(self.params.lock_file))
+        except Exception as e:
```

Comment:
This seems unsafe as if we get any Exception when trying to create the file it's going to continue as if the lock was created successfully

Reply:
I think this `try` block is actually useless, because it catches and just throws whatever it caught with no extra handling. It is just inherited from how `save_to_store()` was written.

I would remove this `try` block, but we might be putting it back anyway (with some enhancements) to address https://github.com/aws-robotics/aws-robomaker-sample-application-objecttracker/issues/15.